src/Resources/translations/SonataPageBundle.de.xliff

```diff
@@ -358,7 +358,7 @@
         <source>sidemenu.link_list_snapshots</source>
         <target>Publikationen</target>
       </trans-unit>
-      <trans-unit id="90" resname="snapshot">
+      <trans-unit id="90" resname="form.group_snapshot">
         <source>snapshot</source>
```

Comment:
I think you should modify the sources for all cases. TBH not sure what is "resname"

Reply:
We can remove "resname" and put them instead of id numbers like SonataAdminBundle.
Should I change in this PR or better to create a PR for all translations SonataPageBundle files?

Reply:
If you can do it in separate PRs it would be nice.

Reply:
Sure, will do that :slightly_smiling_face:

Reply:
Even if we do in a separate PRs, here we should fix the <source> field of translations.